conda_content_trust/common.py

```diff
@@ -433,13 +433,6 @@ def checkformat_hex_key(k):
     if 64 != len(k):
         raise ValueError("Expected a 64-character hex string representing a key value.")
 
-    # Prevent multiple possible representations of keys.  There are security
-    # implications.  For example, we cannot permit two signatures from the
-    # same key -- with the key represented differently -- to count as two
-    # signatures from distinct keys.
-    if k.lower() != k:
-        raise ValueError("Hex representations of keys must use only lowercase.")
-
```

Comment:
Why remove this?

Reply:
Unreachable since checkformat_hex_string() (why not use bytes.fromhex() and see what happens instead?) also checks for lowercase.

Reply:
ah, okay, sounds good